html/cross-origin-opener-policy/reporting/resources/report.py

```diff
@@ -1,26 +1,26 @@
 import json, uuid
 
 def main(request, response):
-    response.headers.set('Cache-Control', 'no-cache, no-store, must-revalidate');
+    response.headers.set(b'Cache-Control', b'no-cache, no-store, must-revalidate')
 
-    key = 0;
-    if 'endpoint' in request.GET:
-        key = uuid.uuid5(uuid.NAMESPACE_OID, request.GET['endpoint']).get_urn()
+    key = 0
+    if b'endpoint' in request.GET:
+        key = uuid.uuid5(uuid.NAMESPACE_OID, request.GET[b'endpoint']).get_urn()
```

Comment:
get_urn() doesn't exist in python3; however, both python2 and python3 have a `urn` property: https://docs.python.org/3/library/uuid.html#uuid.UUID.urn https://docs.python.org/2/library/uuid.html#uuid.UUID.urn

Reply:
Updated code. Thank you!